flathub.json

```diff
@@ -0,0 +1,3 @@
+{
+    "skip-arches": ["i386"]
+}
```

Comment:
Should we also skip the arm and aarch64 builds then?

Reply:
See https://community.metabrainz.org/t/flatpak-version-of-picard-2-0/370699.

Reply:
Thanks for asking them!

(I would have just gone and ask on IRC, but this is better as it will give us a URL we can point to)